### PR TITLE
Moved DefaultAttempts to the config file

### PR DESCRIPTION
**Pull request by alexhurs**

### CODE REVIEW

File: Behavioral.Automation.Selenium/Behavioral.Automation.DemoBindings/AutomationConfig.json [hunk 1]
```diff
@@ -9,5 +9,6 @@
   "BAUTH_PASSWORD": "",
   "BAUTH_IGNORE": "true",
   "BROWSER_BINARY_LOCATION" : "",
-  "UNHANDLED_PROMPT_BEHAVIOR" : ""
+  "UNHANDLED_PROMPT_BEHAVIOR" : "",
+  "DEFAULT_ASSERT_ATTEMPTS" : 30
```
Comment (andrey-brovko) [inline Behavioral.Automation.Selenium/Behavioral.Automation.DemoBindings/AutomationConfig.json:13]:
```suggestion
  "ASSERT_ATTEMPTS" : 30
```

File: Behavioral.Automation.Selenium/Behavioral.Automation/Services/ConfigServiceBase.cs [hunk 2]
```diff
@@ -31,6 +31,8 @@ public static class ConfigServiceBase
         private const string BrowserBinaryLocationString = "BROWSER_BINARY_LOCATION";
```
Comment (andrey-brovko) [inline Behavioral.Automation.Selenium/Behavioral.Automation/Services/ConfigServiceBase.cs:31]: What if we will use `Behavioral.Automation.Configs.ConfigManager`?

File: Behavioral.Automation.Selenium/Behavioral.Automation.DemoBindings/AutomationConfig.json (hunk 1 above)
Comment (alexhurs) [reply Behavioral.Automation.Selenium/Behavioral.Automation.DemoBindings/AutomationConfig.json]: updated

File: Behavioral.Automation.Selenium/Behavioral.Automation/FluentAssertions/Assert.cs [hunk 3]
```diff
@@ -14,7 +12,8 @@ namespace Behavioral.Automation.FluentAssertions
 {
     public static class Assert
     {
-        private const int DefaultAttempts = 30;
+        private static readonly int DefaultAttempts = ConfigServiceBase.DefaultAttempts;
```
[the remark this answers is not in the document]
Comment (alexhurs) [reply Behavioral.Automation.Selenium/Behavioral.Automation/FluentAssertions/Assert.cs]: updated

File: Behavioral.Automation.Selenium/Behavioral.Automation/Services/ConfigServiceBase.cs (hunk 2 above)
Comment (alexhurs) [reply Behavioral.Automation.Selenium/Behavioral.Automation/Services/ConfigServiceBase.cs]: updated

**Review by andrey-brovko — APPROVED** [review]

**Review by ILastovka — APPROVED** [review]

**Pull request closed without merging**